From: NeilBrown <neilb@suse.de>
Subject: kabi fix
Patch-mainline: never
References: bnc#864404

Fix kabi break in 
 patches.fixes/0001-NFSv4-Allow-the-state-manager-to-mark-an-open_owner-.patch

Acked-by: NeilBrown <neilb@suse.de>
Signed-off-by: Neil Brown <neilb@suse.de>

---
 fs/nfs/nfs4_fs.h |    2 ++
 1 file changed, 2 insertions(+)

--- linux-3.0-SLE11-SP3.orig/fs/nfs/nfs4_fs.h
+++ linux-3.0-SLE11-SP3/fs/nfs/nfs4_fs.h
@@ -127,7 +127,9 @@ struct nfs4_state_owner {
 	struct list_head     so_states;
 	struct nfs_seqid_counter so_seqid;
 	struct rpc_sequence  so_sequence;
+#ifndef __GENKSYMS__
 	seqcount_t	     so_reclaim_seqcount;
+#endif
 };
 
 enum {
